internal/saucecloud/cucumber.go

```diff
@@ -29,10 +30,14 @@ func (r *CucumberRunner) RunProject() (int, error) {
 		r.logFrameworkError(err)
 		return exitCode, err
 	}
+	r.setVersions(m)
```

Comment:
Move it before the validation since the deprecation msg inside of validation method also requires framework version.